File: ContactBook.Web/contactbook.vue/src/models/ContactAdress.ts
export default class adresses {
    id:number;
    contactId:number;
    street:string;
    houseNumber:string;
    city:string;
    lat:number;
    lng:number;
    type:AdressType

    constructor (id:number, contactId:number, street:string, houseNumber:string, city:string, type:AdressType, lat:number, lng:number) {
      this.id = id
      this.contactId = contactId
      this.street = street
      this.houseNumber = houseNumber
      this.city = city
      this.type = type
      this.lat = lat
      this.lng = lng
    }
}
export enum AdressType{
    Home,
    Work
}

File: ContactBook.core/Models/ContactAdress.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ContactBook.core
{
    public class ContactAdress
    {
        public int Id { get; set; }
        public int ContactId { get; set; }
        public string Street { get; set; }
        public string HouseNumber { get; set; }
        public string City { get; set; }
        public AdressType Type { get; set; }
        public float lat { get; set; }
        public float lng { get; set; }
        public virtual Contact contact { get; set; }
        
        
    }
    public enum AdressType
    {
        Home ,
        Work ,
    }
}

